clang/test/SemaSYCL/intel-fpga-loops.cpp

```diff
@@ -285,13 +285,13 @@ void zoo() {
   [[intel::max_interleaving(1)]]
   [[intel::loop_coalesce]] for (int i = 0; i != 10; ++i)
       a[i] = 0;
-  [[intel::max_interleaving(1)]]
-  // expected-error@+2 {{duplicate Intel FPGA loop attribute 'max_interleaving'}}
+  [[intel::max_interleaving(1)]] // expected-note {{previous attribute is here}}
+  // expected-error@+2 {{conflicting loop attribute 'max_interleaving'}}
   [[intel::speculated_iterations(1)]]
   [[intel::max_interleaving(0)]] for (int i = 0; i != 10; ++i)
       a[i] = 0;
-  [[intel::speculated_iterations(1)]]
-  // expected-error@+2 {{duplicate Intel FPGA loop attribute 'speculated_iterations'}}
+  [[intel::speculated_iterations(1)]] // expected-note {{previous attribute is here}}
+  // expected-error@+2 {{conflicting loop attribute 'speculated_iterations'}}
   [[intel::loop_coalesce]]
   [[intel::speculated_iterations(2)]] for (int i = 0; i != 10; ++i)
       a[i] = 0;
```

Comment:
Can you please add a test case for the variant I mentioned above?

Reply:
Done